### PR TITLE
config.tf: Bump kvo, kubernetes, console.

### PR DESCRIPTION
**Pull request by yifan-gu**

### CODE REVIEW

**Comment by yifan-gu** [comment]
test upgrade from 1.7.5-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.7.5-tectonic.1 1.7.5-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.7.5-tectonic.1 1.7.5-tectonic.1

File: config.tf [hunk 1]
```diff
@@ -77,7 +77,7 @@ variable "tectonic_container_images" {
     kubednsmasq                  = "gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.5"
     kubedns_sidecar              = "gcr.io/google_containers/k8s-dns-sidecar-amd64:1.14.5"
     kube_version                 = "quay.io/coreos/kube-version:0.1.0"
-    kube_version_operator        = "quay.io/coreos/kube-version-operator:v1.7.5-kvo.12"
+    kube_version_operator        = "quay.io/coreos/kube-version-operator:v1.7.9-kvo.1"
```
Comment (diegs) [inline config.tf]: Does this KVO include the console bump above?

**Comment by yifan-gu** [comment]
test upgrade from 1.7.5-tectonic.1 1.7.5-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.7.5-tectonic.1 1.7.5-tectonic.1

**Pull request merged**